### ISSUE TITLE
arcface(s=64,m=0.5) Accuracy is worse than Softmax

### ISSUE THREAD

**Issue by zhanglaplace** [cavalleria/cavaface.pytorch#60]
train data:
glint360Datasets(https://github.com/deepinsight/insightface/blob/master/recognition/partial_fc/README.md);
1、 First，train baseline model with R50-Softmax
2、Second，Train ArcFace-Softmax by finetune the backbone using Step1 Backbone model . the acc is lower then origin softmax;
![image](https://user-images.githubusercontent.com/12178974/100040839-ffd80b00-2e42-11eb-846a-01a0e84b00f0.png)

**Comment by cavalleria** [comment]
I have not finetuned using arcface, but the result does not seem reasonable.

**Comment by zhanglaplace** [comment]
> I have not finetuned using arcface, but the result does not seem reasonable.
train stretch using arcface has a better performance ?

**Comment by cavalleria** [comment]
training from scratch using arcface is better than softmax. i think properly finetune on softmax with arcface should better softmax
